Steer/DigitizerWorkflow/src/FT0DigitizerSpec.cxx

```diff
@@ -58,10 +59,10 @@ class FITDPLDigitizerTask
       mSimChains.emplace_back(new TChain("o2sim"));
       mSimChains.back()->AddFile(signalfilename.c_str());
     }
-    static constexpr o2::detectors::DetID::ID DETID = o2::detectors::DetID::FT0;
-    if (mID == o2::detectors::DetID::FT0) {
-      mDigitizer.init();
-    }
+    //  static constexpr o2::detectors::DetID::ID DETID = o2::detectors::DetID::FT0;
```

Comment:
I don't see why using some many hardcoded "FT0" strings is better than having a data member 
````
static constexpr o2::detectors::DetID::ID DETID = o2::detectors::DetID::FT0;
````
and using ``DETID.getName()`` whenever used?